.env.example

```diff
@@ -1,2 +1,3 @@
-CLIENT_ID=
-CLIENT_SECRET=
\ No newline at end of file
+USERNAME=MY_USERNAME
```

Comment:
Let's rename this to `USER_ID` because of what we recently run into of Linux refilling USERNAME ENV variable with Linux user's username by default plus it matches with how it's displayed in the UI.